fda-model/metrics/IContainResults.cs

```diff
@@ -3,12 +3,9 @@ namespace metrics
 {
     public interface IContainResults
```

Comment:
forcing properties is good. another alternative is to force the signature for your retrieval - how do you expect people to gather data? getting the two objects you provide via properties and drilling down works, an alternative could be something like a method like double MeanEADByCategory(string category); but that might be too specific for our current situation.